server/routes/public.js

```diff
@@ -27,4 +27,22 @@ router.get('/api/login', async(req, res) => {
     res.send(user);
 });
 
-module.exports = router 
\ No newline at end of file
+module.exports = router 
```

Comment:
Hi Nancy, I would suggest moving lines 35 to 48 before the router is exported and also moving line 34 to the top of the file with the other handler import statement. I would also recommend renaming the handler to be `accountHandler` as opposed to `accountCreationHandler` to avoid confusion.

Reply:
Hi Nancy, did you push your changes? I still see these lines below the router export statement. Additionally, there is no need to create another router (line 32) as it has already been created above) in line 5.

Reply:
Hi Nancy, I see the changes now, Thank you for pushing them!